inc/class-wpseo-utils.php

```diff
@@ -973,6 +973,23 @@ public static function get_user_locale() {
 		return get_locale();
 	}
 
+	/**
+	 * Returns the full name for the user's language.
+	 *
+	 * @since 5.9
+	 *
+	 * @returns string The language name.
```

Comment:
Not sure we should use @returns or @return, I see a bit of inconsistency across php files